test/e2e/framework/util.go

```diff
@@ -443,15 +394,6 @@ func ProviderIs(providers ...string) bool {
 	return false
 }
 
-func ClusterMonitoringModeIs(monitoringModes ...string) bool {
```

Comment:
SkipUnlessClusterMonitoringModeIs is only caller as https://cs.k8s.io/?q=ClusterMonitoringModeIs&i=nope&files=&repos=kubernetes/kubernetes and that is removed as the above.